### ISSUE TITLE
Linker fails

### ISSUE THREAD

**Issue by SoFiA-Admin** [SoFiA-Admin/SoFiA#94]
As mentioned at the telecon, SoFiA currently fails at the linker stage with the following error:

    --- SoFiA: Merging detections ---
    Traceback (most recent call last):
      File "sofia_pipeline.py", line 271, in <module>
        objects, mask = linker.link_objects(np_Cube, mask, **Parameters['merge'])
      File "linker.pyx", line 96, in linker.link_objects (src/linker/linker.cpp:1289)
    TypeError: link_objects() takes at least 3 positional arguments (2 given)

**Comment by RussellJurek** [comment]
I figured it out. I hadn't updated the sofia_pipeline.py script to pass in an empty list of objects. I made some changes to the linker.pyx script in anticipation of the `processing large data cubes' work. The linker module is now setup to add new objects to the end of an existing array of objects. Previously it could only provide a new list, which would need to be manually combined with any existing objects.
